include/ensmallen_bits/utility/indicators/epsilon.hpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file epsilon.hpp
+ * @author Rahul Ganesh Prabhu
+ * @author Nanubala Gnana Sai
+ *
+ * Epsilon indicator
+ * A binary quality indicator that is capable of detecting whether one
+ * approximation set is better than another.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef ENSMALLEN_INDICATORS_EPSILON_HPP
+#define ENSMALLEN_INDICATORS_EPSILON_HPP
+
+namespace ens {
+
+/**
+ * The epsilon indicator is a binary quality indicator that was proposed by
+ * Zitzler et. al. in response to the observation that quality indicators could
+ * only give information that one approximation set is not worse than another.
+ * The binary epsilon indicator, however, is capable of detecting if an
+ * approximation is better than another.
```

Comment:
This description is too generic as all the indicators proposed in the paper satisfy this property.